deeppavlov/dataset_iterators/document_bert_ner_iterator.py

```diff
@@ -49,19 +49,21 @@ class DocumentBertNerIterator(DataLearningIterator):
     def __init__(self,
                  data: Dict[str, List[Tuple[Any, Any]]],
                  bert_tokenizer_vocab_file: str,
+                 do_lower_case: bool = False,
```

Comment:
Feels like a relief

Reply:
You bet!￼￼ <img src="https://emojipedia-us.s3.dualstack.us-west-1.amazonaws.com/thumbs/160/google/146/party-popper_1f389.png" width="20" height="20" />